walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,8 +14,59 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    long duration = request.getDuration();
+    int[] minutes = new int[1441];
```

Comment:
Why not 1440?
Also, I'd prefer to have this written as 24*60 or alternatively just write a short comment that says this is the number of minutes in a day, so that it's more readable :).

Reply:
+1. Or even create a constant named MINUTES_IN_ONE_DAY = 1440 and use this constant later (that's how you don't even need comments)

Reply:
Thanks, I fixed with using constant.